metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/publish/DefaultLogPublisher.java

```diff
@@ -102,14 +111,67 @@ protected void printLog(List<Meter> meters) {
     DefaultPublishModel model = factory.createDefaultPublishModel();
 
     printThreadPoolMetrics(model, sb);
-
+    printOsLog(factory.getTree(), sb);
     printConsumerLog(model, sb);
     printProducerLog(model, sb);
     printEdgeLog(model, sb);
 
     LOGGER.info(sb.toString());
   }
 
+  protected void printOsLog(MeasurementTree tree, StringBuilder sb) {
+    if (!DynamicPropertyFactory.getInstance()
+        .getBooleanProperty(METRICS_OS_ENABLED, false)
+        .get()) {
+      return;
+    }
+    MeasurementNode OsNode = tree.findChild(OsStatisticsMeter.OS_STATISTICS_NAME);
+    if (OsNode != null && !OsNode.getMeasurements().isEmpty()) {
+      MeasurementNode netNode = OsNode.findChild("net");
+      MeasurementNode cpuNode = OsNode.findChild("cpu");
+      if (cpuNode != null && !cpuNode.getMeasurements().isEmpty()) {
+        double rate = cpuNode.findChild("allCpu", OsCpuMeter.tagCpuRate.value()).summary();
+        String ratePercentStr = "calc ...";
```

Comment:
why "calc ..."?
why NumberFormat ?

Reply:
just give a tip... NumberFormat have removed

Reply:
I have removed the tip